pkg/apis/servicecatalog/types.go

```diff
@@ -191,7 +191,7 @@ type ServiceClass struct {
 
 	// Plans is the list of ServicePlans for this ServiceClass.  All
 	// ServiceClasses have at least one ServicePlan.
-	Plans []ServicePlan
+	Plans []Plan
```

Comment:
we'll need to be able to traverse from the serviceclass to the plan quickly - can the controller (or is it "owner"?) property on serviceclass help here?